tools/backup/README-BACKUP.md

```diff
@@ -1,3 +1,49 @@
 # Backing up Neo4j Containers
 
+##Create Secret
```

Comment:
I appreciate you writing docs with this; but these markdown docs have been deprecated & removed from the repo.  In order that our docs get deployed to the website correctly in the right format, backup documentation should go into this asciidoc file here:  https://github.com/neo4j-contrib/neo4j-helm/blob/master/doc/docs/modules/ROOT/pages/backup.adoc

If you could please revise this, and add the content that you've got that asciidoc file, it would ensure that people can see it and refer to it.